pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -250,19 +259,11 @@ in {
     # IMPORTANT: Always use a tagged release candidate or commits from the
     # zfs-<version>-staging branch, because this is tested by the OpenZFS
     # maintainers.
-    version = "2.1.10-staging-2023-03-15";
-    rev = "a5c469c5f380b09705ad0bee15e2ca7a5f78213c";
+    version = "zfs-2.1.11-staging-2023-04-13";
```

Comment:
I'm curious, why are you prefixing the version here?

Reply:
ah